### ISSUE TITLE
no valid dependencies for jdk, 14.* 😭

### ISSUE THREAD

**Issue by benleb** [paketo-buildpacks/bellsoft-liberica#59]
Hello,
thanks for providing the 15.* (see #14)! But at least @JeremyTwiggs and I now miss the 14.* :( Can we please get it back, for a few weeks maybe? :D 

```
[INFO]     [creator]     Paketo BellSoft Liberica Buildpack 5.2.0
[INFO]     [creator]       https://github.com/paketo-buildpacks/bellsoft-liberica
[INFO]     [creator]       Build Configuration:
[INFO]     [creator]         $BP_JVM_VERSION              14.*            the Java version
[INFO]     [creator]       Launch Configuration:
[INFO]     [creator]         $BPL_JVM_HEAD_ROOM           0               the headroom in memory calculation
[INFO]     [creator]         $BPL_JVM_LOADED_CLASS_COUNT  35% of classes  the number of loaded classes in memory calculation
[INFO]     [creator]         $BPL_JVM_THREAD_COUNT        250             the number of threads in memory calculation
[INFO]     [creator]         $JAVA_TOOL_OPTIONS                           the JVM launch flags
[INFO]     [creator]       No valid JRE available, providing matching JDK instead. Using a JDK at runtime has security implications.
[INFO]     [creator]
[INFO]     [creator]     Paketo BellSoft Liberica Buildpack 5.2.0
[INFO]     [creator]       unable to find dependency
[INFO]     [creator]       no valid dependencies for jdk, 14.*, and io.buildpacks.stacks.bionic in [(jdk, 8.0.275, [io.buildpacks.stacks.bionic org.cloudfoundry.stacks.cflinuxfs3]) (jre, 8.0.275, [io.buildpacks.stacks.bionic org.cloudfoundry.stacks.cflinuxfs3]) (jdk, 11.0.9, [io.buildpacks.stacks.bionic org.cloudfoundry.stacks.cflinuxfs3]) (jre, 11.0.9, [io.buildpacks.stacks.bionic org.cloudfoundry.stacks.cflinuxfs3]) (jdk, 15.0.1, [io.buildpacks.stacks.bionic org.cloudfoundry.stacks.cflinuxfs3]) (jre, 15.0.1, [io.buildpacks.stacks.bionic org.cloudfoundry.stacks.cflinuxfs3]) (jvmkill, 1.16.0, [io.buildpacks.stacks.bionic org.cloudfoundry.stacks.cflinuxfs3])]
[INFO]     [creator]     ERROR: failed to build: exit status 1
```

Thanks!!

**Comment by nebhale** [comment]
The buildpacks, going back to the inception of Cloud Foundry, have a policy of not shipping unsupportable versions of dependencies.  In this case, the OpenJDK community describes the support windows of releases, and they’re quite clear that for non LTS versions, a version becomes unsupported (as in we can no longer upstream fixes for CVEs) one month after the release of the next version.  To facilitate this, the .0 release of a version comes out “off-cycle” a month earlier than the quarterly release schedule would dictate

In the case of the 14 to 15 transition, the buildpack starting shipping **both** Java 14 and 15 with [4.0.0 on September 23rd](https://github.com/paketo-buildpacks/bellsoft-liberica/tree/v4.0.0).  With the release of 15.0.1 (along with Java 8 and 11 releases) we removed 14 in [5.0.0 on October 22](https://github.com/paketo-buildpacks/bellsoft-liberica/tree/v5.0.0 ).  This means that for four weeks we shipped both 14 and 15 giving the OpenJDK-designed window for migration.

This rapid and mandatory upgrade cycle is why both we and the OpenJDK maintainers recommend against going to production on non-LTS versions and instead only using them for testing.
